readme.md

```diff
@@ -18,11 +18,11 @@ The following languages are currently being translated, but we can
 <!-- start language-table -->
 - [Indonesian](https://crowdin.com/project/electron/id)
 - [Türkçe (Turkish)](https://crowdin.com/project/electron/tr)
-- [中文 (Chinese Simplified)](https://crowdin.com/project/electron/zh-CN)
 - [Español (Spanish)](https://crowdin.com/project/electron/es-ES)
```

Comment:
Can you revert changes to the readme here? These changes will happen in the build.

Reply:
Sure.